### PR TITLE
Update to cpp-driver 2.13 and latest Bincrafters templates

### PR DESCRIPTION
**Pull request by sourcedelica**

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -5,78 +5,84 @@
...
     default_options = "shared=False", "fPIC=True"
 
-    # Custom attributes for Bincrafters recipe conventions
     source_subfolder = "source_subfolder"
-    build_subfolder = "build_subfolder"
 
     requires = (
-        "libuv/1.25.0@bincrafters/stable",
+        "libuv/1.27.0@bincrafters/stable",
         "OpenSSL/1.0.2o@conan/stable",
```
Comment (Croydon) [inline conanfile.py]: Please update OpenSSL

File: conanfile.py [hunk 2]
```diff
@@ -5,78 +5,84 @@
...
+            tools.replace_in_file(os.path.join(self.source_subfolder, "cmake", "modules", "CppDriver.cmake"),
+                                  "if(WIN32)\n      install",
+                                  "if(WIN32 AND NOT MINGW)\n      install")
+            tools.replace_in_file(os.path.join(self.source_subfolder, "cmake", "modules", "CppDriver.cmake"),
+                                  'elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")',
+                                  'elseif("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")\n' +
+                                  'set(CASS_LIBS ${CASS_LIBS} iphlpapi psapi wsock32 crypt32 ws2_32 userenv)')
         cmake = self.configure_cmake()
         cmake.build()
+        cmake.install()
```
Comment (Croydon) [inline conanfile.py]: `install` should be executed in package, not in build

File: conanfile.py [hunk 3]
```diff
@@ -5,78 +5,84 @@
...
         cmake.definitions['LIBUV_ROOT_DIR'] = self.deps_cpp_info["libuv"].rootpath
+        if self.settings.os == 'Windows' and self.options.shared:
+            if self.settings.compiler == 'gcc':
+                libuv_library = os.path.join(self.deps_cpp_info["libuv"].rootpath, "bin", "libuv.dll")
+            else:
+                libuv_library = os.path.join(self.deps_cpp_info["libuv"].rootpath, "lib", "libuv.dll.lib")
+            cmake.definitions['LIBUV_LIBRARY'] = libuv_library
         cmake.definitions['OPENSSL_ROOT_DIR'] = self.deps_cpp_info["OpenSSL"].rootpath
         if self.settings.os != 'Windows':
             cmake.definitions['CMAKE_POSITION_INDEPENDENT_CODE'] = self.options.fPIC
```
Comment (Croydon) [inline conanfile.py]: This isn't needed anymore for a while now

File: conanfile.py [hunk 4]
```diff
@@ -5,78 +5,84 @@
...
     exports_sources = ["CMakeLists.txt"]
     generators = "cmake"
 
-    # Options may need to change depending on the packaged library.
     settings = "os", "arch", "compiler", "build_type"
     options = {"shared": [True, False], "fPIC": [True, False]}
     default_options = "shared=False", "fPIC=True"
 
-    # Custom attributes for Bincrafters recipe conventions
     source_subfolder = "source_subfolder"
```
Comment (Croydon) [inline conanfile.py]: please rename to `_source_subfolder`

**Review by Croydon — CHANGES_REQUESTED** [review]
Please update some outdated stuff

many of the things can be automatically updated via [bincrafters-conventions](https://github.com/bincrafters/bincrafters-conventions)

**Comment by Croydon** [comment]
I also think that the name cassandra is misleading as it is a driver for it, not the database itself

File: conanfile.py (hunk 2 above)
Comment (sourcedelica) [reply conanfile.py]: done

File: conanfile.py (hunk 3 above)
Comment (sourcedelica) [reply conanfile.py]: done

File: conanfile.py (hunk 4 above)
Comment (sourcedelica) [reply conanfile.py]: done

File: conanfile.py (hunk 1 above)
Comment (sourcedelica) [reply conanfile.py]: done

**Comment by sourcedelica** [comment]
bincrafters-conventions run.

How about `conan-cassandra-driver`?

**Comment by uilianries** [comment]
Hi everyone!

So, let's update to `conan-cassandra-driver` ?

**Comment by sourcedelica** [comment]
That is fine with me

**Comment by Croydon** [comment]
Are there other cassadra drivers for C/C++ out there or is this the only (known) one?

**Comment by sourcedelica** [comment]
This is the only C/C++ driver that I know about - it is the official one provided by the developers of Cassandra (Datastax).

**Comment by Croydon** [comment]
Renamed and added missing attributes via https://github.com/bincrafters/conan-cassandra-driver/commit/434b25b3e657525e70be9529b6c0d7c0e49e7d56

**Pull request closed without merging**